markdown/metadata.go

```diff
@@ -8,8 +8,15 @@ import (
 
 // Metadata is document metadata in the "front matter" of a Markdown document.
 type Metadata struct {
-	Title                       string `yaml:"title"`
-	IgnoreDisconnectedPageCheck bool   `yaml:"ignoreDisconnectedPageCheck"`
+	IgnoreDisconnectedPageCheck bool `yaml:"ignoreDisconnectedPageCheck"`
+
+	Title         string   `yaml:"title"`
```

Comment:
I think the docsite already does some inference for the title based on the first h1 on the page - maybe we should leave this out to avoid odd cases?

Reply:
If this is the case I think it would be nice if there was a way to override an inferred title. I'd rather have title explicitly defined in frontmatter than rely on people remembering to start with an H1 tag.

Reply:
oh nevermind, I just realized Title has always been here and is used as the preferred title before falling back this meta.Title  😅